### PR TITLE
fixed modules not beeing added to vsproj

### LINKED ISSUE #36384: "modules" subdirectory is not added to VSProj
**Godot version:**
f5bb6d6a7c6d225b15a8022a8511e3918fd02048

**OS/device including version:**
Win10

**Issue description:**
Seems like files in "modules" subdirectory does not being added to the generated VisualStudio solution:

image

Which prevent their debugging and force a user to manually add them.

**Steps to reproduce:**

Type `scons p=windows vsproj=yes` and open a resulted solution.

### PR DESCRIPTION
**Pull request by HaSa1002**
fixes #36384

### CODE REVIEW

**Comment by pouleyKetchoupp** [comment]
Thanks for the workaround! This fix does the job, but it seems it causes each module to link against all previous module sources when `vsproj` is enabled. I ended up with a bunch of 500MB lib files after a full build.

Maybe we could use 2 separate variables `modules_sources` for all modules and `module_sources` for the current one to avoid this problem?

**Comment by HaSa1002** [comment]
Okay then it was a bug, that my hard drive was full. I have an idea how to fix it, but don't know how fast I can implement it.

**Comment by HaSa1002** [comment]
Okay @pouleyKetchoupp could you retry it? It shouldn't longer pollute the module libs with all modules sources

**Comment by HaSa1002** [comment]
> Maybe we could use 2 separate variables `modules_sources` for all modules and `module_sources` for the current one to avoid this problem?

Yeah kinda, but this isn't possible because of https://github.com/godotengine/godot/blob/c5f71ecc6a60e5a92a1f711a1368e051d4824b25/modules/SCsub#L31-L33

**Comment by pouleyKetchoupp** [comment]
@HaSa1002 Yes it's working fine, thanks!

@akien-mga Could you please review this fix? It's simple and it would be good to have on master for contributors working with Visual Studio :)

File: modules/SCsub [hunk 1]
```diff
@@ -10,7 +10,7 @@ Export("env_modules")
 
 # Header with MODULE_*_ENABLED defines.
 env.CommandNoCache("modules_enabled.gen.h", Value(env.module_list), modules_builders.generate_modules_enabled)
-
+vs_sources = []
```
Comment (akien-mga) [inline modules/SCsub:13]: The separation line should be kept for readability.

Comment (HaSa1002) [reply modules/SCsub:14]: done

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Note to self: not relevant for the 3.2 branch, unless #35982 is cherrypicked (but I don't plan to).

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**